src/viking.js

```diff
@@ -1,14 +1,61 @@
 // Soldier
-class Soldier {}
+class Soldier {
+  constructor(health, strength) {
+    this.health = 300;
+    this.strength = 150;
+  }
```

Comment:
vc fez o health e o strength "hardcoded". Pra que ele fique genérico vc poderia fazer:

this.health = health;
this.strength = strength; 👍